### PR TITLE
Tweak board description

### PR DESCRIPTION
**Pull request by JessamyT**
Attempt to increase parsability

### CODE REVIEW

**Review by andf-viam — APPROVED** [review]
LGTM, assuming line 34's `CPUs` --> `SBCs` is intentional. I think it is and makes sense!

**Comment by JessamyT** [comment]
Thank you, yes it is intentional but will wait for @sguequierre to confirm that it's accurate!

File: docs/components/board/_index.md
Comment (sguequierre) [inline docs/components/board/_index.md:30]: Not sure if it's ideal to use architecture in this way when we have a lot of other stuff about architecture in the more traditional programming sense-- unfortunately I don't have a better idea though 😆 TBH I don't know if splitting this out into bullets is ideal either as the model information has more on this as well. Maybe it's more key info for users than I'm thinking though? Really your call

File: docs/components/board/_index.md
Comment (sguequierre) [inline docs/components/board/_index.md:28]: Hmmm-- sorry to be a bother here but I think splitting this out makes the sentence flow a bit more awkward as it's supposed to be kind of cause and effect!

Comment (sguequierre) [reply docs/components/board/_index.md:28]: Up to you if you want to keep or not, LGTM either way

File: docs/components/board/_index.md
Comment (sguequierre) [inline docs/components/board/_index.md:34]: Again nit picky but re. CPU - SBC I think it is likely if using a PWM that they also might be using peripheral or something? Hard to phrase

Comment (JessamyT) [reply docs/components/board/_index.md:34]: I don't think I understand this comment sorry--can you rephrase?

File: docs/components/board/_index.md
Comment (JessamyT) [reply docs/components/board/_index.md:30]: I think the bullets help a lot because the original paragraph is hard to parse, and the bullets quickly show that there are 3 possible scenarios. Instead of calling them "architectures," do you think it would make sense to replace this sentence with
```suggestion
A board can be:
```
and then change the bullets to be singular (i.e. "A single-board computer....", "A GPIO peripheral device..." etc?

File: docs/components/board/_index.md
Comment (JessamyT) [inline docs/components/board/_index.md:35]:
```suggestion
- A single-board computer (SBC) with GPIO pins and a CPU capable of running `viam-server`.
- A GPIO peripheral device that must connect to an external computer.
- A PWM peripheral device that must connect to an SBC that has GPIO pins.
```

File: docs/components/board/_index.md
Comment (JessamyT) [inline docs/components/board/_index.md:35]:
```suggestion
- A single-board computer (SBC) with GPIO pins and a CPU capable of running `viam-server`.
- A GPIO peripheral device that must connect to an external computer.
- A PWM peripheral device that must connect to an SBC that has a CPU and GPIO pins.
```

**Comment by JessamyT** [comment]
merging per slack convo

**Pull request merged**
